spec/swagger_helper.rb

```diff
@@ -9,12 +9,12 @@
     'v1/swagger.json' => {
```

Comment:
Is it still at `swagger.json`, or now should be `openapi.json`?

Reply:
It should be openapi.json as per https://platform-docs.cloud.paas.upshift.redhat.com/rest/requirements.html

Reply:
So.. this needs updating, was my point :)